src/styles/blocks/stars.scss

```
@@ -0,0 +1,20 @@
+.stars {
+  &__flex {
+    display: flex;
+    gap: 4px;
+
+    @for $stars from 1 through 5 {
+      &--#{$stars} :nth-child(-n+#{$stars}) {
+      background-image: url(../images/star-active.svg);
+      }
+    }
+  }
```

```suggestion
.stars {
  display: flex;
  gap: 4px;

  @for $stars from 1 through 5 {
    &--#{$stars} :nth-child(-n+#{$stars}) {
    background-image: url(../images/star-active.svg);
  }
```
i.e. remove stars-flex at all (and wrapper in the HTML file)
modificator should be `stars--4` instead of `stars__flex--4`